lib/mvi/service.rb

```diff
@@ -141,29 +141,24 @@ def build_exception(key, error)
       )
     end
 
-    def mvi_error_handler(user_identity, e)
+    def mvi_error_handler(user_identity, e, source = '')
       case e
       when MVI::Errors::DuplicateRecords
-        log_console_and_sentry('MVI Duplicate Record', :warn)
+        log_message_to_sentry('MVI Duplicate Record', :warn)
       when MVI::Errors::RecordNotFound
-        log_console_and_sentry('MVI Record Not Found')
+        Rails.logger.info('MVI Record Not Found')
       when MVI::Errors::InvalidRequestError
         # NOTE: ICN based lookups do not return RecordNotFound. They return InvalidRequestError
         if user_identity.mhv_icn.present?
-          log_console_and_sentry('MVI Invalid Request (Possible RecordNotFound)', :error)
+          log_message_to_sentry("MVI Invalid Request (Possible RecordNotFound) #{source}", :error)
         else
-          log_console_and_sentry('MVI Invalid Request', :error)
+          log_message_to_sentry('MVI Invalid Request', :error)
         end
       when MVI::Errors::FailedRequestError
-        log_console_and_sentry('MVI Failed Request', :error)
+        log_message_to_sentry('MVI Failed Request', :error)
       end
     end
 
-    def log_console_and_sentry(message, sentry_classification = nil)
```

Comment:
this method was pointless because `log_message_to_sentry` already writes to the rails log.